apps/docs/content/guides/auth/social-login/auth-workos.mdx

```diff
@@ -1,172 +1,129 @@
 ---
 id: 'auth-workos'
-title: 'Login with WorkOS'
-description: 'Add WorkOS OAuth to your Supabase project'
+title: 'SSO and Social Login with WorkOS'
+description: 'Add WorkOS AuthKit to your Supabase project'
 ---
 
-To enable WorkOS Auth for your project, you need to set up WorkOS OAuth application and add the application credentials to your Supabase Dashboard.
+## Choose which authentication method is right for you
 
-## Overview
+There are two ways to add authentication to your Supabase using WorkOS: 
 
-In this guide, we will cover how to use Supabase OAuth with WorkOS to implement Single-Sign-On(SSO).
+### [Option 1](#option-1-using-workos-as-a-provider-via-supabase-auth): Using WorkOS as a provider via Supabase auth 
 
-The procedure consists of five broad steps:
+This method uses WorkOS as a provider you configure in your Supabase dashboard. You then use Supabase's auth code to connect to WorkOS. 
 
-- Create a new organization from your WorkOS Dashboard.
-- Obtain the `Client ID` from the Configuration tab and configure redirect URI.
-- Obtain the `WorkOS Secret` from the credentials tab.
-- Connect a WorkOS Supported Identity Provider
-- Add your WorkOS credentials into your Supabase project
+### [Option 2](#option-2-using-authkit): Using AuthKit
 
-## Create a WorkOS organization
+[WorkOS AuthKit](https://authkit.com) is the world's best login box. AuthKit provides email/password authentication and social login by default. Once AuthKit is installed, you can configure and deploy additional SSO providers via the WorkOS dashboard.
 
-Log in to the dashboard and hop over to the Organizations tab to create and organization
-![Create an Organization](/docs/img/guides/auth-workos/workos-create-organization.png)
+## Option 1: Using WorkOS as a provider via Supabase auth 
+
+### Step 1. Create a WorkOS organization
 
-## Obtain the client ID and configure redirect URI
+Log in to the WorkOS dashboard and visit the Organizations tab to create an organization
```

Comment:
Should we mention that you can do this programmatically via the API?

Reply:
Yeah good call - I'll add.

Reply:
Added in [54d86d0](https://github.com/zackproser-workos/supabase/pull/1/commits/54d86d0d34e45c7f55496e723c2fa235fc0cf083)